modules/config.xqm

```diff
@@ -62,13 +62,24 @@ declare variable $config:FRUS_METADATA := collection($config:FRUS_METADATA_COL);
 
 declare variable $config:FRUS_CODE_TABLES_COL := "/db/apps/frus/code-tables";
 
+declare variable $config:x-forwarded-host        := request:get-header("X-Forwarded-Host");
+declare variable $config:x-forwarded-proto       := request:get-header("X-Forwarded-Proto");
+declare variable $config:is-proxied              := not(empty($config:x-forwarded-host));
+declare variable $config:is-proxied-path-to-root := request:get-context-path() || substring-after($config:is-proxied, "gov/") || "/";
```

Comment:
Please confirm `$config:is-proxied` is the variable you mean here; it will be a boolean.

Reply:
I have deleted the variable with https://github.com/HistoryAtState/hsg-shell/pull/371/commits/d42474bbe2703a50fa1f45f772467dbfdf509dc2 . The PR can be reviewed and merged now.